resources/keywords/import_provider_account.resource

```diff
@@ -223,8 +224,15 @@ Provider Account Imported Successfully Using CLI
     Wait Until Keyword Succeeds    4x    5s    Describe Provider Account Using OC CLI
 
 Describe Provider Account Using OC CLI
```

Comment:
Can you please explain what 'describe provider account' means?

Reply:
This step will execute "Oc describe "command for that resource and fetches detailed information about it. This information is used to find the status of the resource (imported provider account).